<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rulesMenu</class>
 <widget class="QWidget" name="rulesMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 24pt &quot;Segoe UI&quot;;

color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Rules MENU</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="healing_checkBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 127);
color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>healing allowed</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="buffing_checkBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>buffing allowed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Confirm_Button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>340</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(0, 255, 0);
font: 900 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="PP_checkBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Use PP System </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
